jbpm/jbpm-flow/src/main/resources/org/jbpm/marshalling/jbpmmessages.proto

```diff
@@ -176,7 +176,7 @@ message Variable {
     optional string name = 1;
     // the actual object reference
     optional int32 strategy_index = 2;
-    optional bytes value = 3;
+    optional string value = 3;
```

Comment:
@rhtevan @xiezhang7 @ruromero 
I have changed this attribute datatype to "String" from "bytes" to store the JSON string. It impacted existing infinispan and file persistence and I had handled it by implementing the corresponding methods to read and write Object from and to JSON. Please let me know if you have any suggestions.

Reply:
@swati-kale As per previous discussion, adding a new field would be a better strategy to provide compatibility to existing ISPN implementation. In addition, need to refactor existing code to allow large pieces of 'copy-&-paste'. Thanks.

Reply:
@rhtevan , I have pushed the above changes. Thanks.

Reply:
In case you want to test with Quarkus, here is the example,
https://github.com/swati-kale/kogito-examples/pull/1

Reply:
@rhtevan, I have pushed the renaming, formatting, and integration test changes. Thanks.

Reply:
@swati-kale I'm checking it.